examples/README.md

```diff
@@ -1,42 +1,14 @@
-# Example using Differential Privacy library
 
-In this directory, we give a simple example of how to use the Python Differential
-Privacy library.
+## Curated list of Tutorials and Sample code for PyDP!
 
-## Zoo Animals
 
-There are around 182 animals at Farmer Alex's zoo. Every day, Alex feeds
-the animals as many carrots as they desire. The animals record how many carrots
-they have eaten per day. For this particular day, the number of carrots eaten
-can be seen in `animals_and_carrots.csv`.
+### Tutorials
 
-At the end of each day, Alex often asks aggregate questions about how many
-carrots everyone ate. For example, he wants to know how many carrots are eaten
-each day, so he knows how many to order the next day. The animals are fearful
-that Alex will use the data against their best interest. For example, Alex could
-get rid of the animals who eat the most carrots!
+#### Carrots Demo
```

Comment:
number it (maybe later when we have the restautant demo)

Reply:
we have "error handling" as example 2 so better number them as - 1 and 2 respectively.